services/user/Homepage/ui/src/apps/contacts/types.ts

```diff
@@ -0,0 +1,16 @@
+import { Account } from "@/lib/zod/Account";
+import { z } from "zod";
+
+export const LocalContact = z.object({
+    account: Account,
+    nickname: z.string().optional(),
+    email: z.string().optional(),
+    phone: z.string().optional(),
+});
+
+export const ProcessedContact = LocalContact.extend({
+    avatar: z.string(),
+});
```

Comment:
What would you think about switching to a convention for zod objects and starting them with lowercase `z`. So it would be `zLocalContact` and `zProcessedContact`. These aren't TypeScript types so it doesn't quite make sense to start them with a capital letter. And because the start with `z`, we'll always know we're dealing with a zod object.

Then we don't have to suffix TS types with `Type` as you're doing below. Types follow normal convention and start with a capital letter.

Thoughts?

Reply:
I like it! Adopted with these types, can do another PR to pick up the pre-existing zod schemas later.